app/models/drawing.rb

```diff
@@ -4,6 +4,7 @@ class Drawing < ActiveRecord::Base
 
   validates :image, presence: true
   validates :status, presence: true
+  validates_inclusion_of :image_consent, in: [true, false]
```

Comment:
Good choice, server-side validation should always be in place regardless of client-side also for tests sake :dancer: